packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -99,24 +112,37 @@ export default class QuanticRefineModalContent extends LightningElement {
     this.headless = getHeadlessBundle(this.engineId);
     this.searchStatus = this.headless.buildSearchStatus(engine);
     this.breadcrumbManager = this.headless.buildBreadcrumbManager(engine);
+    this.initializeSortOptions();
+
     this.unsubscribeSearchStatus = this.searchStatus.subscribe(() =>
-      this.gatherFacets()
+      this.initializeFacets()
     );
     this.unsubscribeBreadcrumbManager = this.breadcrumbManager.subscribe(() =>
       this.updateHasActiveFilters()
     );
   };
 
   /**
-   * Gather all facets registered in the Quantic store.
+   * Initializes all facets registered in the Quantic store.
    * @returns {void}
    */
-  gatherFacets() {
+  initializeFacets() {
     if (!this.hasFacets) {
       this.data = getAllFacetsFromStore(this.engineId);
     }
   }
 
+  /**
+   * Initializes all sort options registered in the Quantic store.
+   * @returns {void}
+   */
+  initializeSortOptions() {
+    this.sortData = getAllSortOptionsFromStore(this.engineId);
+    if (this.sortData?.length) {
+      this.sortOptionsInitialized = true;
```

Comment:
The name isn't really representative if this variable depends on the `sortData.length`.

What are you trying to do here is it something like this perhaps?

```
get hasSortOptions() {
  return this.sortData?.length > 0;
}
```

I would use this `getter` instead of the `sortOptionsInitialized` if I understand correctly this determine if you should be displaying the sort section of the refined modal?

Reply:
yeah that is what I was going for, I wanted to avoid the getter since you mentioned in one of Mehdi's PR that getters should be used mostly when its to be used in the html templates

Reply:
But you ARE using it in the template :D 

`<template lwc:if={sortOptionsInitialized}>`

Reply:
oh in that sense -_- ok I thought you meant like `this. sortOptionsInitialized = this.hasSortOptions` :D 

That makes more sense XD